variables.tf

```diff
@@ -121,8 +121,7 @@ variable "common_tags" {
 }
 
 variable "asp_rg" {
-  description = "Name of the resource group where the asp specified in asp_name resides"
-  default     = "null"
```

Comment:
there's specific handling in the arm template for this value, I would leave it in as is...